### PR TITLE
Release/0.1.18

### PR DESCRIPTION
**Pull request by tkruse**
@wjwwood : This pr is the version update ready to be released after https://github.com/vcstools/vcstools/issues/146

Travis failure because new vcstools not released yet.

Feel free to look over the last merged commits.
I'll go ahead with in a week else.

### CODE REVIEW

File: stdeb.cfg [hunk 1]
```diff
@@ -3,5 +3,5 @@ Depends: subversion, mercurial, git-core, bzr, python-yaml, python-vcstools (>=
 Depends3: subversion, mercurial, git-core, bzr, python3-yaml, python3-vcstools (>= 0.1.38)
 Conflicts: python3-wstool
 Conflicts3: python-wstool
-Suite: oneiric precise quantal raring saucy trusty utopic vivid wily xenial yakkety zesty artful bionic wheezy jessie stretch buster
+Suite: oneiric precise quantal raring saucy trusty utopic vivid wily xenial yakkety zesty artful bionic cosmic disco wheezy jessie stretch buster
```
Comment (wjwwood) [inline stdeb.cfg]: This won't work right now, could you remove those until we have updated our infrastructure to allow it?

Comment (tkruse) [reply stdeb.cfg]: removed.

**Comment by tkruse** [comment]
I made a release to testpypi at https://test.pypi.org/project/wstool/. I assume that @wjwwood will again make a combined release.

**Comment by wjwwood** [comment]
Sorry, I've been on vacation. I'll get to this as soon as I can.

**Comment by tkruse** [comment]
@wjwwood : reminding ping

**Comment by wjwwood** [comment]
@tkruse sorry for the delay, but while trying to release this I ran into an issue with the `setup.py`. Unless I comment this line (https://github.com/vcstools/wstool/blob/42ce038de31a8d456982c2bbd1f31725484e73a5/setup.py#L125) I get this error when running `python3 setup.py sdist bdist_wheel`:

```
...
copying build/scripts-3.6/wstool -> build/bdist.linux-x86_64/wheel/wstool-0.1.18.data/scripts
changing mode of build/bdist.linux-x86_64/wheel/wstool-0.1.18.data/scripts/wstool to 775
Traceback (most recent call last):
  File "setup.py", line 140, in <module>
    license="BSD")
  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/lib/python3/dist-packages/wheel/bdist_wheel.py", line 257, in run
    self.distinfo_dir)
  File "/usr/lib/python3/dist-packages/wheel/bdist_wheel.py", line 427, in egg2dist
    distribution=self.distribution)
  File "/usr/lib/python3/dist-packages/wheel/metadata.py", line 177, in pkginfo_to_dict
    new_requirements = sorted(convert_requirements(requirements))
  File "/usr/lib/python3/dist-packages/wheel/metadata.py", line 234, in convert_requirements
    parsed_requirement = pkg_resources.Requirement.parse(req)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2988, in parse
    req, = parse_requirements(s)
ValueError: not enough values to unpack (expected 1, got 0)
```

Some details about the machine I was running on:

```
% python3 -m pip --version
pip 19.2.3 from /usr/local/lib/python3.6/dist-packages/pip (python 3.6)

% python3 -m pip freeze --all | grep setuptools
setuptools==40.8.0

% python3 --version
Python 3.6.8

% lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic

% which python3
/usr/bin/python3
```

I poked at it for a while, but I'm not exactly sure what needs to change to avoid breaking how the requirements are being used. If you have some time to look at that I'd appreciate it.

**Comment by tkruse** [comment]
My system seems fine, travis also seems fine.
My system wheel:
```
python3 -m pip freeze --all | grep wheel
wheel==0.33.5
```
I guess the comment lines can just be removed if all else fails, but maybe you want to give it another shot updating your wheel version first?

**Comment by tkruse** [comment]
I added the command python3 setup.py sdist bdist_wheel to tox.ini being run on travis, to check it is sane on an up-to-date environment

**Pull request merged**